jekyll/_includes/snippets/ide-extension-banner.adoc

```diff
@@ -0,0 +1,3 @@
+CircleCI has created a VS Code extension that reduces the time to create, modify, and troubleshoot configuration files through real-time syntax validation, highlighting, and autocomplete suggestions. Authenticating the extension with your CircleCI account will provide you with status alerts as well as the ability to validate your configuration by testing the pipeline locally before you push your code. 
```

Comment:
Same thing as above. 
Can I also double check where this message will appear? 
If it's not in a config-specific part of the docs, I'd probably highlight the pipeline management feature by reversing the order in which the two are presented. Something like this:

> CircleCI has created a VS Code extension that allows you to visualise and manage CircleCI pipelines directly from your IDE, as well as be notified of workflow status changes.
It also reduces the time to create, modify and troubleshoot CircleCI configuration files through real-time syntax validation, highlighting, contextual documentation and autocomplete suggestions.

Reply:
This is a reusable text snippet, which I created when I thought all the pages would be the same format (asciidoc or markdown) but it's mixed content types, so I'll probably get rid of this snippet and put it on the page directly.